src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+  const uniqueRobotValues = [...new Set(robotValues)];
```

Comment:
There's no need to store Set as an array, if you use it only for comparing length, you can use `size` property